admin/handlers.php

```diff
@@ -188,8 +193,11 @@ function gigpress_error_checking($context) {
 				$errors['artist_name'] = __("You must enter an artist name.", "gigpress");
 			if($_POST['show_venue_id'] == 'new' && empty($_POST['venue_name']))
 				$errors['venue_name'] = __("You must enter a venue name.", "gigpress");
-			if($_POST['show_venue_id'] == 'new' && empty($_POST['venue_city']))
-				$errors['venue_city'] = __("You must enter a city.", "gigpress");
+			if ( true === $venue_city_required ) {
```

Comment:
Ideally we already know that `$venue_city_required` is a boolean at this point, so by doing `if ( $venue_city_required ) {}` you're doing the same comparison without using the `true`. I'd ideally go that route, or to be sure that you cover all bool cases, I'd use `if ( ! empty( $venue_city_required ) )`

Reply:
Clarification question:
A 'false' value would be empty?
So if `$venue_city_required` is false, then `! empty( $venue_city_required )` would be false as well?

Reply:
Right, `empty` would be `true` if the parameter is: 0, '', array(), false, null -> https://www.php.net/manual/en/function.empty.php

`! empty( $venue_city_required)` would consider when it's not empty (essentially false, or the values I listed above), or in other words "true".

Reply:
With this change, people using the filter can use `return 1;` instead of true, and it'll be taken into account. So it's more "contemplative" if you do it this way.

Normally for the other plugins we have a function called "tribe_is_truthy()". We don't have that here, so `empty` would be the more "broad" solution. 

Another option/solution would be to cast the `$venue_city_required` and do the `if` with that directly. Casting would be something like:

`$venue_city_required = (bool) apply_filters( 'gigpress_venue_city_required', true );`

Reply:
Fantastic, thanks for the explanation!